server/services/inductionService.ts

```diff
@@ -24,4 +25,14 @@ export default class InductionService {
       throw error
     }
   }
+
+  async createInduction(prisonNumber: string, createInductionDto: CreateInductionDto, token: string): Promise<unknown> {
+    try {
+      const request = toCreateInductionRequest(createInductionDto)
+      return await this.educationAndWorkPlanClient.createInduction(prisonNumber, request, token)
+    } catch (error) {
+      logger.error(`Error creating Induction within the Education And Work Plan API: ${JSON.stringify(error)}`)
```

Comment:
```
logger.error('Error creating Induction within the Education And Work Plan API', error)
```
would be better, as you'll get a proper stack trace in App Insights (rather than a weird json/string version of the error - bearing in mind the error type will be `Error` rather than something that can always be serialised in this way)

(though I note with interest that that is exactly what I've done in the `getInduction` method! 😬 - can you change them both please?)

(also, I can see we have a few of these JSON.stringify style in PLP UI - I'll raise a PR shortly to change those 👍 )